admin_manual/configuration/server/security/ransomware-protection.rst

```diff
@@ -51,19 +82,24 @@ A combination of Ransomware prevention and protection reduces risks to a minimum
 Other Elements of Ransomware Protection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-==================== ============================================== ================================================
-Name                 Command                                        Description
-==================== ============================================== ================================================
-Ransomguard Scanner  ``occ ransomguard:scan <timestamp> <user>``    A command to scan the ownCloud database for
-                                                                    changes in order to discover anomalies in a 
-                                                                    user's account and their origin. It enables an 
-                                                                    administrator to determine the point in time
-                                                                    where undesired actions happened as a
-                                                                    prerequisite for restoration.
-Ransomguard Restorer ``occ ransomguard:restore <timestamp> <user>`` A command for administrators to revert all
-                                                                    operations in a user account that occurred after
-                                                                    a certain point in time.
-==================== ============================================== ================================================
+=============================== ============================================== ========================================================
+Name                            Command (if applicable)                        Description
+=============================== ============================================== ========================================================
```

Comment:
Indentation issues in this paragraph or is it just displayed badly? The section looked well-structured before.

Reply:
"ransomguard:lock/unlock" commands are missing here?

Reply:
I'll add the commands in. Wrt the formatting, it's likely just how GitHub displays it.